src/components/TimeRangePicker/TimeRangePickerBody.js

```diff
@@ -0,0 +1,77 @@
+import Calendar from 'react-calendar';
+import { TimeRangeList } from "./TimeRangeList";
+import "./index.css";
+import clsx from "clsx";
+import { useState } from 'react';
+import { MiniCalendarIcon } from '../../icons/mini-calendar';
+import { CloseIconX } from '../../icons/close-x';
+
+
+export const TimeRangePickerBody = ({ isOpen, closePicker }) => {
+
+  let [showCalendar, setShowCalendar] = useState(false);
+    
+  return (
+    <div className={clsx("time-range-picker-body flex justify-center cursor-auto w-max bg-gray-50 h-96 absolute rounded-sm border border-gray-300 z-50 shadow-lg shadow-gray-200", isOpen && 'active')}>
+      <div className={clsx("calendar-wrapper absolute p-2 bg-gray-50 rounded-sm border border-gray-300 shadow-lg shadow-gray-200", showCalendar && 'active')}>
```

Comment:
Make the most of `clsx`
```suggestion
      <div className={clsx("calendar-wrapper absolute p-2 bg-gray-50 rounded-sm border border-gray-300 shadow-lg shadow-gray-200", {active: showCalendar})}>
```